website/docs/docs/dbt-versions/versionless-cloud.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Upgrade to \"Keep on latest version\" in dbt Cloud"
+sidebar_label: "Upgrade to \"Keep on latest version\" "
+description: "Learn how to go versionless in dbt Cloud. You never have to perform an upgrade again. Plus, you'll be able to access new features and enhancements as soon as they become available. "
+---
+
+dbt Cloud is going versionless. Soon, your environments and jobs will always run on the latest version of dbt.
+
+This will require you to make one final update to your current jobs and environments. When that's done, you never have to think about managing, coordinating, or upgrading dbt versions again.
+
+Move your environments and jobs to "Keep on latest version" to get all the functionality in the latest versions of dbt Core &mdash; and more! &mdash; along with access to the new features and fixes as soon as they’re released.
+
+## Tips for upgrading {#upgrade-tips}
+
+If you regularly develop in dbt Cloud and this is your first time trying “Keep on latest version,” dbt Labs recommends that you start by [overriding your dbt version in development](/docs/dbt-versions/upgrade-dbt-version-in-cloud#override-dbt-version). Then, launch the IDE or Cloud CLI and do your development work as usual. Everything should work as you expect. If you experience anything working differently than from how you’d expect, run a few models in development to ensure everything is looking as you’d expect.
```

Comment:
Hiya @nghi-ly 

This is a non-blocker, but do you think it would be worthwhile giving a small explanation as to why we recommend new users to override their dbt version?

Kind Regards